cl_dll/hud_watermark.cpp

```diff
@@ -22,9 +24,20 @@ int CHudWatermark::VidInit()
 
 int CHudWatermark::Draw(float time)
 {
-	if (refresh_draw_until || (draw_until > gHUD.m_flTime + 15.0f)) {
```

Comment:
How about:
```cpp
float duration = 15.0f;
if (hud_watermark->value == 0)
    duration = 5.0f;
```

Reply:
Yes, that's a good point. I'll implement the floater when I'm free tomorrow